manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
```

Comment:
Sure!
Let me do all the migration in this PR.
On Sun, Nov 15, 2015 at 12:32 AM Sean Lip <notifications@github.com> wrote:

> In manifest.json
> <https://github.com/oppia/oppia/pull/1127#discussion_r44864020>:
>
> > @@ -0,0 +1,144 @@
> > +{
> > +  "dependencies": {
>
> Ah, ok. Would it be difficult to start the migration in this PR, so that
> we can delete the corresponding stuff in install_third_party? We don't have
> to remove everything; I'm just looking for an intermediate state where a
> dependency is specified either here or there, but not both.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oppia/oppia/pull/1127/files#r44864020>.
>